example/default.nix

```diff
@@ -0,0 +1,43 @@
+{ pkgs ? import <nixpkgs> { } }:
+let
+  mvn2nix = import
+    (fetchTarball "https://github.com/fzakaria/mvn2nix/archive/add-pom.tar.gz")
+    { };
+  buildMavenRepository = mvn2nix.buildMavenRepository;
+  mavenRepository =
+    buildMavenRepository { dependencies = import ./dependencies.nix; };
+  inherit (pkgs) lib stdenv jdk11_headless maven makeWrapper;
+  inherit (stdenv) mkDerivation;
+in mkDerivation rec {
+  pname = "mvn2nix";
+  version = "0.1";
+  name = "${pname}-${version}";
```

Comment:
Would be great to actually create another friendly lib that can consume the _pom.xml_ file to read this.